samples/package.json

```diff
@@ -1,20 +1,25 @@
 {
   "name": "nodejs-security-center-samples",
-  "main": "quickstart.js",
   "private": true,
   "engines": {
     "node": ">=8"
   },
   "scripts": {
-    "test": "mocha system-test"
+    "test": "mocha system-test/*.js --timeout 6000000",
+    "check": "gts check",
+    "clean": "gts clean",
+    "fix": "gts fix"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@google-cloud/security-center": "^0.3.0"
+    "@google-cloud/security-center": "^0.3.0",
+    "gts": "^0.9.0"
```

Comment:
We shouldn't have `gts` in here.  Linting and fixing is handled in the parent directory.